exercism/diamond/diamond.js

```diff
@@ -0,0 +1,49 @@
+var Diamond = function() {};
+
+Diamond.prototype.makeDiamond = function(input) {
+  let letter = input.toUpperCase()
+let alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+let diamondLimit = []
```

Comment:
`diamondLimit` doesn't express what this array is.  You could rename it to `diamondLetters`.